contractcourt/chain_watcher.go

```diff
@@ -446,11 +461,16 @@ func (c *chainWatcher) handleUnknownLocalState(
 	// Next, we'll derive our script that includes the revocation base for
 	// the remote party allowing them to claim this output before the CSV
 	// delay if we breach.
+	localAuxLeaf := fn.MapOption(
+		func(l lnwallet.CommitAuxLeaves) input.AuxTapLeaf {
+			return l.LocalAuxLeaf
+		},
+	)(auxLeaves)
```

Comment:
map followed by flatten is just [`ChainOption`](https://github.com/lightningnetwork/lnd/blob/master/fn/option.go#L136-L148)